transports/upgrades/asb-9.1toasbs-1.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Azure Service Bus Transport Legacy 9.1 to Azure Service Bus 1.0
+summary: Azure Service Bus legacy to Azure Service Bus transport upgrade guide.
+reviewed: 2020-02-26
+component: ASBS
+related:
+ - transports/azure-service-bus
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 7
+ ---
+
+## Migrating from Azure Service Bus (Legacy) to Azure Service Bus Transport
+
+Endpoints using the Azure Service Bus Transport can be run side-by-side with endpoints using the legacy Azure Service Bus transport as long as the [backwards compatibility](/transports/azure-service-bus/compatibility.md) requirements are met.
+
+## API Differences
+
+There are differences between the API for the legacy Azure Service Bus and the Azure Service Bus transports.
+
+### Max lock duration
+
+Azure Service Bus set message lock duration (`MaxLockDuration`) to a maximum of 5 minutes by default. This value has been chosen for enhanced reliability and to support the most common use-cases. One of the reasons to set lock duration to the maximum is to avoid the need to reconfigure the setting when the default time of 30 seconds is not sufficient for processing and which would cause `LockLostException` exceptions. 
+
+The most common use case for NServiceBus endpoints is to process messages without the need to manage for lock expiration. Therefore the default values for LockDuration and Prefetch in the Azure Service Bus transport have been selected to ensure no messages lose their locks when processing is longer than 30 seconds. If processing is successful, the message will be acknowledged and completed. In case the message fails to process, it will go through recoverability. Only when there's an abnormal shutdown behaviour (e.g. such as an endpoint crashing) would take up to 5 minutes for the message to re-appear.
```

Comment:
```suggestion
The most common use case for NServiceBus endpoints is to process messages without the need to manage for lock expiration. Therefore the default values for `LockDuration` and `Prefetch` in the Azure Service Bus transport have been selected to ensure no messages lose their locks when processing is longer than 30 seconds. If processing is successful, the message will be acknowledged and completed. In case the message fails to process, it will go through recoverability. Only when there's an abnormal shutdown behaviour (e.g. such as an endpoint crashing) would take up to 5 minutes for the message to re-appear.
```